lib/jxl/transfer_functions-inl.h

```diff
@@ -165,6 +165,13 @@ class TF_709 {
 // Perceptual Quantization
 class TF_PQ {
  public:
+  explicit TF_PQ(float display_intensity_target) {
+    display_scaling_factor_to_10000_nits_ =
+        display_intensity_target * (1.0f / 10000.0f);
+    display_scaling_factor_from_10000_nits_ =
+        1.0f / display_scaling_factor_to_10000_nits_;
+  }
```

Comment:
These two fields could be made const by initialising them as:

```suggestion
  explicit TF_PQ(float display_intensity_target)
      : display_scaling_factor_to_10000_nits_(display_intensity_target *
                                              (1.0f / 10000.0f)),
        display_scaling_factor_from_10000_nits_(
            1.0f / display_scaling_factor_to_10000_nits_) {}
```

(Also, for the latter, if we are going to compute a division anyway then perhaps we might as well make it `10000.f / display_intensity_target`.)

Reply:
Wouldn't making the const break the assignment operator?

Not sure if it necessarily matters, but it might.